wondrous-app/components/NavigationProgress/styles.tsx

```diff
@@ -0,0 +1,31 @@
+import styled from 'styled-components';
+
+export const ProgressBarWrapper = styled.div`
```

Comment:
May we use MUI Grid here?

Reply:
We could I guess but why? it's a very basic div

Reply:
It's up to you but I find that wrapper components like this becomes more obvious to the reader and it's reusable.
https://mui.com/system/getting-started/usage/#why-use-mui-system

Though there's some performance tradeoff
https://mui.com/system/getting-started/usage/#when-to-use-mui-system